ddtrace/internal/telemetry/writer.py

```diff
@@ -216,26 +228,29 @@ def _create_telemetry_request(self, payload, payload_type, sequence_id):
             "request_type": payload_type,
         }
 
-    def _restart(self):
+    def _fork_writer(self):
         # type: () -> None
-        if self.status == ServiceStatus.RUNNING:
-            self.disable()
-        self.enable()
+        self._forked = True
+        # Avoid sending duplicate events.
+        # Queued events should be sent in the main process.
+        self._flush_events_queue()
```

Comment:
This hook is called in the child process, right?

If so then aren't we then flushing the duplication events here?